src/logtracing/Logger.py

```diff
@@ -1,20 +1,107 @@
 import os
+import sys
+import json
+import time
+import psutil
+import platform
 import traceback
 from typing import Union
-from Types import PrepareStackTrace, CodeLine, ErrorStack, OsVars, PythonVars
+from Types import CodeLine, ErrorStack, PythonVars, OsVars, PrepareStackTrace
 
 
 class Logger:
     def __init__(self, flow: str):
-      self._flow = flow;
-      self.prepare_stack_trace: PrepareStackTrace = traceback.extract_stack
-      self.err_stack = None;
-      self.os_vars: Union[OsVars, None] = None;
-      self.python_vars: Union[PythonVars, None] = None;
-      self.env_vars: Union[os.environ, None] = None;
-      self.extra_vars = {};
-      self.code_lines_limit: int = 5;
+        if not flow:
+            raise Exception('Flow argument is missing')
+
+        self._flow = flow
+        self.prepare_stack_trace: PrepareStackTrace = traceback.extract_stack
+        self.err_stack: Union[ErrorStack, None] = None
+        self.os_vars: Union[OsVars, None] = None
+        self.python_vars: Union[PythonVars, None] = None
+        self.env_vars: Union[os.environ, None] = None
+        self.extra_vars = {}
+        self.code_lines_limit: int = 5
 
     @property
     def flow(self):
```

Comment:
Would you consider add this type of return?
```suggestion
    def flow(self) -> str:
```

Reply:
@renebarrera1022 Can you look at this? After that, of changes are needed or not, pls close the comment.

Reply:
Changed it. Thank you!